### PR TITLE
Added new issue related to ANF and FSLogix

### PR DESCRIPTION
**Pull request by igorpag**

### CODE REVIEW

File: checklists/avd_checklist.en.json [hunk 1]
```diff
@@ -643,11 +643,20 @@
...
       "guid": "b2d1215a-e114-4ba3-9df5-85ecdcd9bd3b",
       "severity": "Medium",
       "link": "https://docs.microsoft.com/fslogix/cloud-cache-configuration-reference"
     },
+    {
+      "category": "Storage",
+      "subcategory": "FSLogix",
+      "text": "If Azure NetApp Files storage is used, check AD Site name setting in the AD Connection.",
+      "description": "An Active Directory Site should be created for the Azure virtual network environment where ANF subnet will be created, and that site name should be specified in the ANF connection property when executing the join procedure as explained in the reference article.",
+      "guid": "b2d1215a-e114-4ba3-9df5-85ecdcd9bd3b",
```
Comment (erjosito) [inline checklists/avd_checklist.en.json:656]: It looks like this GUID is the same as in line 647?

Comment (igorpag) [reply checklists/avd_checklist.en.json:656]: Ouch! I used copy&paste the previous element to create a template and then modified. Very bad, now I understand it. How to fix it? Removing only that column value?

Comment (igorpag) [reply checklists/avd_checklist.en.json:656]: @erjosito when creating/adding a new element, how to deal with "guid" attribute? We should include with empty "" value or not including it at all? How and when that value is generated?

Comment (erjosito) [reply checklists/avd_checklist.en.json:656]: @igorpag The way I do it, I add the new checklist item in the spreadsheet leaving the GUID column blank, and then I export the spreadsheet to JSON (to the avd_checklist.en.json file). That will generate a new GUID. Alternatively you can just generate a new GUID offline (for example in Linux with the command `uuidgen`), and then put this in the JSON.

For this PR I would suggest to generate a new random GUID (for example with `uuidgen`) and commit it to your fork igorpag:main.

Comment (igorpag) [reply checklists/avd_checklist.en.json]: Done!

**Review by erjosito — APPROVED** [review]

**Pull request merged**